slowly-changing-lookup-cache/README.md

```diff
@@ -0,0 +1,70 @@
+# slowly-changing-lookup-cache
+An implementation of "Pattern: Slowly-changing lookup cache" pattern of [Guide to common Cloud Dataflow use-case patterns](https://cloud.google.com/blog/big-data/2017/06/guide-to-common-cloud-dataflow-use-case-patterns-part-1).
+
+## What is this
+
+* Reading a streaming data from Cloud PubSub as `mainInput` for pipeline. 
+* Enriching `mainInput` data with `sideInput`.
+    - In this example, both inputs are `string`, we merging these two and showing as an log.
+* `sideInput` can be any external resources. For the convenience, reading a text file from Cloud Storage.
+* `sideInput` is a slow changing data. We are updating this every `N` seconds. 
+
+<img src="console.png" width="600">
+
+## How to Run
+
+0. Please substitute appropriate values
```

Comment:
I think we might as well instruct users to set these values to environment variables. 
In that case, you don't need `${}` here.
Please refer to a template I proposed in #5 .